pkg/karmadactl/cordon.go

```diff
@@ -52,6 +52,7 @@ func NewCmdCordon(karmadaConfig KarmadaConfig, parentCommand string) *cobra.Comm
 
 	flags := cmd.Flags()
 	opts.GlobalCommandOptions.AddFlags(flags)
+	flags.BoolVar(&opts.DryRun, "dry-run", false, "Run the command in dry-run mode, without making any server requests.")
```

Comment:
suggest to blank line that make it clearer.

```suggestion

	flags.BoolVar(&opts.DryRun, "dry-run", false, "Run the command in dry-run mode, without making any server requests.")
```